nin/dasBoot/TextureInput.js

```diff
@@ -0,0 +1,8 @@
+(function(NIN) {
+  'use strict';
+
+  class TextureInput extends NIN.Input {
```

Comment:
It is subclassed just for type safety. JS doesn't often check types, but we can. This lets us for instance display different UI widgets for different types of inputs down the line, if we want.